frontend/packages/topology/src/utils/topology-utils.ts

```diff
@@ -44,13 +44,28 @@ export const getCheDecoratorData = (consoleLinks: K8sResourceKind[]): CheDecorat
   };
 };
 
+const getFullGitURL = (gitUrl: GitUrlParse.GitUrl, branch?: string) => {
+  const baseUrl = `https://${gitUrl.source}/${gitUrl.owner}/${gitUrl.name}`;
```

Comment:
Based on the learnings in #8006 we should use gitUrl.resource here instead, or?

```suggestion
  const baseUrl = `https://${gitUrl.resource}/${gitUrl.owner}/${gitUrl.name}`;
```

Reply:
Almost forgot this :P thanks.